src/CallHandler.js

```diff
@@ -402,18 +403,30 @@ function _onAction(payload) {
 }
 
 function _startCallApp(roomId, type) {
+    const room = MatrixClientPeg.get().getRoom(roomId);
+    if (!room) {
+        console.error("Attempted to start conference call widget in unknown room: " + roomId);
+        return;
+    }
+
     dis.dispatch({
         action: 'appsDrawer',
         show: true,
     });
 
-    const room = MatrixClientPeg.get().getRoom(roomId);
-    if (!room) {
-        console.error("Attempted to start conference call widget in unknown room: " + roomId);
+    const currentRoomWidgets = WidgetUtils.getRoomWidgets(room);
+
+    if (WidgetEchoStore.roomHasPendingWidgetsOfType(room, currentRoomWidgets, 'jitsi')) {
+        const ErrorDialog = sdk.getComponent("dialogs.ErrorDialog");
+
+        Modal.createTrackedDialog('Already have pending Jitsi Widget', '', ErrorDialog, {
```

Comment:
Using an internal tracking ID that is not clearly related to what is shown in the modal might be confusing.